plugins/modules/pnp_intent.py

```diff
@@ -611,36 +629,41 @@ def get_have(self):
                             self.msg = "Installation mode must be in \
                             INSTALL mode to upgrade the image. Current mode is\
                             {0}".format(install_mode)
+                            self.log(str(self.msg), "CRITICAL")
                             self.status = "failed"
                             return self
 
                         have["image_id"] = image_list[0].get("imageUuid")
-                        self.log("Image Id: " + str(have["image_id"]))
+                        self.log("Image Id is {0}".format(str(have["image_id"])), "INFO")
```

Comment:
print image name as well.
                        self.log("Image ID for the image '{0}':  {1}".format(image_name, str(have["image_id"])), "INFO")

Reply:
Resolved